src/opcodes/tests/test_stack.cairo

```diff
@@ -291,6 +291,36 @@ fn test_op_2rot_insufficient_items() {
     utils::check_dstack_size(ref engine, 5);
 }
 
+#[test]
+fn test_max_stack_size_more() {
+    let mut program: ByteArray = "";
+    let op_1_string = "OP_1 ";
+    let mut index: u64 = 0;
+    while index < 1001 {
+        program.append(@op_1_string);
+        index += 1;
+    };
+
+    let mut engine = utils::test_compile_and_run_err(program, Error::SCRIPT_STACK_SIZE_EXCEEDED);
+
+    utils::check_dstack_size(ref engine, 1001);
+}
+
+#[test]
+fn test_max_stack_size_equal() {
```

Comment:
Both tests are adding 1001 elements at the moment

Reply:
hmm, i see what is happening here